tfjs-layers/src/engine/container_test.ts

```diff
@@ -385,9 +385,9 @@ describeMathCPUAndGPU('Container', () => {
 
   it('getLayer works by index', () => {
     const [container, layers] = createSimpleTwoLayerContainer();
-    expect(container.getLayer(0)).toEqual(layers[0]);
-    expect(container.getLayer(1)).toEqual(layers[1]);
-    expect(container.getLayer(2)).toEqual(layers[2]);
+    expect(container.getLayer(null, 0)).toEqual(layers[0]);
+    expect(container.getLayer(null, 1)).toEqual(layers[1]);
+    expect(container.getLayer(null, 2)).toEqual(layers[2]);
   });
```

Comment:
Nit: I didn't really make it clear in my last review. I think we should test both cases. I've added the other case here.
```suggestion
  });

  it('getLayer works by index when index is the first argument', () => {
    expect(container.getLayer(0)).toEqual(layers[0]);
    expect(container.getLayer(1)).toEqual(layers[1]);
    expect(container.getLayer(2)).toEqual(layers[2]);
  });
```